packaging/nuget/nservicebus.ravendb.nuspec

```diff
@@ -16,7 +16,7 @@
     <tags>nservicebus servicebus msmq cqrs publish subscribe ravendb</tags>
     <dependencies>
       <dependency id="NServiceBus" version="[6.0.0, 7.0.0)" />
-      <dependency id="RavenDB.Client" version="[3.0.30000, 3.1.0)" />
+      <dependency id="RavenDB.Client" version="[3.5.0, 3.6.0)" />
```

Comment:
I was keeping the approach to the version range consistent. If the @Particular/ravendb-persistence-maintainers want to support a wider range I am ok with that